include/envoy/runtime/runtime.h

```diff
@@ -165,6 +165,11 @@ class Loader {
    * @param values the values to merge
    */
   virtual void mergeValues(const std::unordered_map<std::string, std::string>& values) PURE;
+
+  /**
+   * @return RandomGenerator& the random generator.
+   */
+  virtual RandomGenerator& random() PURE;
```

Comment:
Do we really need to add this to this interface? I guess it's OK, but can't you just pass the random generator to where you need it? It should be in the filter config factory? This is what we do elsewhere.

Reply:
Thought that I'd have to pass the `RandomGenerator` instance a long way, but I was wrong. I've fixed this in 54c6ac0.